src/nvim/event/process.c

```diff
@@ -267,11 +267,7 @@ static void children_kill_cb(uv_timer_t *handle)
     }
     uint64_t elapsed = now - proc->stopped_time;
 
-    if (!proc->term_sent && elapsed >= TERM_TIMEOUT) {
-      ILOG("Sending SIGTERM to pid %d", proc->pid);
-      uv_kill(proc->pid, SIGTERM);
-      proc->term_sent = true;
-    } else if (elapsed >= KILL_TIMEOUT) {
+    if (elapsed >= KILL_TIMEOUT) {
```

Comment:
Oh.. should I then maybe change it back to be nanoseconds (also better in case somebody is patching this already)?